webpack.config.js

```diff
@@ -97,10 +105,15 @@ module.exports = {
             {from: 'intl', to: 'js'}
         ]),
         new webpack.optimize.UglifyJsPlugin({
+            mangle: true,
```

Comment:
Oh, thanks. I didn't realize it was on by default because when I added this it reduced the file size significantly.  Apparently I did something else at the same time, so I'll double check and remove it if it's unnecessary.